product-tours/tour-tf-keras/Tour-With-TensorFlow-and-Keras.ipynb

```diff
@@ -0,0 +1,2217 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "QXGSKS1u0n8x",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "# Tour With TensorFlow and Keras"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "DO7yawFVxVIt",
+    "tags": [
+     "installation"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! pip install neptune-client==0.4.128 neptune-contrib==0.24.9 tensorflow==2.3.1 scikit-plot==0.3.7 matplotlib==3.3.3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "DO7yawFVxVIt",
+    "tags": [
+     "installation",
+     "library_updates"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! pip install neptune-client neptune-contrib tensorflow scikit-plot matplotlib --upgrade"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 1: Import Libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import hashlib\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import neptune\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import tensorflow as tf\n",
+    "from neptunecontrib.api import log_table\n",
+    "from neptunecontrib.monitoring.keras import NeptuneMonitor\n",
+    "from scikitplot.metrics import plot_roc, plot_precision_recall"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 2: Select project"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "neptune.init('shared/tour-with-tf-keras-tests',\n",
```

Comment:
There should be a cell bellow this linking/showing how to get your own token

Reply:
added.